### ISSUE TITLE
This page needs link to VirtualKey

### ISSUE THREAD

**Issue by bubba9999** [MicrosoftDocs/winrt-api#346]
This page should contain a link to:
https://docs.microsoft.com/en-us/uwp/api/windows.system.virtualkey

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: e2d934fb-8d19-bb5b-0731-b96cf9a7d3b7
* Version Independent ID: 43144078-70fa-95cf-5d1b-46bd41d99746
* Content: [KeyRoutedEventArgs Class (Windows.UI.Xaml.Input) - UWP app developer](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.input.keyroutedeventargs)
* Content Source: [https://cpubwin.visualstudio.com/DefaultCollection/windows-uwp/_git/winrt-api-build?path=%2fwinrt-api-build%2fwindows.ui.xaml.input.keyroutedeventargs.yml&version=GBlive&_a=contents](https://cpubwin.visualstudio.com/DefaultCollection/windows-uwp/_git/winrt-api-build?path=%2fwinrt-api-build%2fwindows.ui.xaml.input.keyroutedeventargs.yml&version=GBlive&_a=contents)
* Service: **unspecified**
* GitHub Login: @wdg-dev-content
* Microsoft Alias: **wdg-dev-content**

**Comment by bubba9999** [comment]
The documentation Section also fails to mention the most important fields of  KeyRoutedEventArgs, the Key field and how its of type "VirtualKey" and ohh by the way here are the values for this enum if you click the link.

**Comment by Karl-Bridge-Microsoft** [comment]
Thanks for the feedback! I've updated the topic you mention (along with a couple of other related) with links to https://docs.microsoft.com/en-us/uwp/api/windows.system.virtualkey.

RE. your second comment: The list of properties for this class are listed in the page after the examples and in the TOC on the left. So there are two ways to get to the standalone Key property topic from the class page. I have also updated the description of the Key property.
